packages/plugin-dotenv/src/index.ts

```diff
@@ -0,0 +1,208 @@
+import { readFileSync } from 'fs';
+import dotenv from 'dotenv-defaults';
+import type { Compiler, RspackPluginInstance, Target } from '@rspack/core';
+import { interpolate, isMainThreadElectron } from './utils';
+
+export interface DotenvPluginOptions {
+  /**
+   * The path to your environment variables.
+   * @default './.env'.
+   */
+  path?: string | undefined;
```

Comment:
Since you have already marked the `path` field as optional, the `undefined` looks a little redundant.

Reply:
Same for the rest type definition.

Reply:
thanks for spotting it, done